src/timestamp.ts

```diff
@@ -28,12 +18,20 @@ export interface TimestampExtended {
  * @public
  * @category BSONType
  */
-export class Timestamp extends LongWithoutOverridesClass {
+export class Timestamp extends BSONValue {
```

Comment:
There is a lot of functionality going away without obvious replacements. Should `ts1.equals(ts2)` keep working or should users manually re-implement it? `ts1.greaterThan(ts2)`? `ts1.isZero()`? Users who access the two components of a Timestamp probably use `ts.high, ts.low` or `ts.getHighBits(), ts.getLowBits()` or something similar, should that keep working?

Is it really intended to break user code this strongly? I can absolutely that it does not make sense to have e.g. a `ts1.multiply(ts2)` API, but there’s a lot of things on `Long` that really do make sense for `Timestamp` objects as well.